### DISCUSSION TITLE
default config.yaml

### DISCUSSION THREAD

**Discussion by LoayJabre** [MannLabs/alphadia#404] [Questions / Support] [answered]
I'm using alphadia on our cluster, and it would be great if there was a default config.yaml file that people can use as a starting point. Does this exist anywhere?

**Comment by mschwoer** [comment, marked as answer]
Hi,

it does [here](https://github.com/MannLabs/alphadia/blob/main/alphadia/constants/default.yaml).

This is taken as a base config, everything you pass by `--config` (or `--config-dict`) will overwrite the values in here (there's a nice printout at the beginning of each run). In this sense, you have already been using it as a starting point ;-)
 
From the next release on, the actual config that is being used (i.e. default + user settings) will be written to the output as a `yaml`.

best,
Magnus

**Reply by LoayJabre** [reply]
amazing thank you!!
